prog/vm/nexus.rb

```diff
@@ -334,6 +334,14 @@ def write_params_json
       amount: vm.cores
     )
 
+    BillingRecord.create_with_id(
+      project_id: project.id,
+      resource_id: vm.id,
```

Comment:
Just an idea. Should we create individual billing records for each VmStorageVolume? They may use different storage backends and have varying prices in the future.

Reply:
Makes sense.